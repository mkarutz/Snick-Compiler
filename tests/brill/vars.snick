proc main ()
    int x;
    float y;
    bool z;
    
    write x;
    write " ";
    write y;
    write " ";
    write z;
    write "\n";
    
    x := 1;
    y := 1.0;
    z := true;
    
    write x;
    write " ";
    write y;
    write " ";
    write z;
    write "\n";
end
