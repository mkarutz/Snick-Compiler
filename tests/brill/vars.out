0 0.000000 false
1 1.000000 true
